### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by foxhound87** [mobxjs/mobx-react-devtools#11]
Hello, I got issues loading the devtools with server side rendering.

I'm using react-router v2 following this guide:
https://github.com/reactjs/react-router-tutorial/blob/start/lessons/13-server-rendering.md

If I remove the devtools, the page is rendered correctly.

```
> react-project@1.0.0 server:prod /react-project
> NODE_ENV=production node ./server.bundle.js

/react-project/node_modules/mobx-react-devtools/index.js:6
!function(){"use strict";function o(){for(var e=[],t=0;t<arguments.length;t++){var n=arguments[t];if(n){var r=typeof n;if("string"===r||"number"===r)e.push(n);else if(Array.isArray(n))e.push(o.apply(null,n));else if("object"===r)for(var a in n)i.call(n,a)&&n[a]&&e.push(a)}}return e.join(" ")}var i={}.hasOwnProperty;"undefined"!=typeof e&&e.exports?e.exports=o:(n=[],r=function(){return o}.apply(t,n),!(void 0!==r&&(e.exports=r)))}()},function(e,t,o){var n=o(9);"string"==typeof n&&(n=[[e.id,n,""]]);o(17)(n,{});n.locals&&(e.exports=n.locals)},function(e,t,o){t=e.exports=o(10)(),t.push([e.id,'.wSDk9Hyhl-CKUJEuLIr5y{position:fixed;height:26px;background-color:#fff;color:rgba(0,0,0,.8);border-radius:0 0 2px 2px;border-style:solid;border-width:0 1px 1px;border-color:rgba(0,0,0,.1);z-index:65000;font-family:Helvetica,sans-serif;display:flex;padding:0 5px}._3HHdqO8CEZraYlCdQ--7a9{color:rgba(0,0,0,.6);text-align:center;text-decorat

ReferenceError: window is not defined
    at /react-project/node_modules/mobx-react-devtools/index.js:6:8347
    at /react-project/node_modules/mobx-react-devtools/index.js:6:8279
    at e.exports (/react-project/node_modules/mobx-react-devtools/index.js:6:8565)
    at Object.<anonymous> (/react-project/node_modules/mobx-react-devtools/index.js:6:507)
    at t (/react-project/node_modules/mobx-react-devtools/index.js:1:733)
    at Object.e (/react-project/node_modules/mobx-react-devtools/index.js:1:9037)
    at t (/react-project/node_modules/mobx-react-devtools/index.js:1:733)
    at Object.t.exports (/react-project/node_modules/mobx-react-devtools/index.js:1:1851)
    at t (/react-project/node_modules/mobx-react-devtools/index.js:1:733)
    at e.__esModule.default (/react-project/node_modules/mobx-react-devtools/index.js:1:820)

```

**Comment by mweststrate** [comment]
Thanks for reporting. DevTools will have no use server side, but I'll make sure they will just do nothing, silently.

**Comment by mweststrate** [comment]
Should be fixed by version `4.0.2`

**Comment by foxhound87** [comment]
It works! thank you
